data/blog/electron-11-0.md

```diff
@@ -0,0 +1,72 @@
+
+We can't wait to see what you build with them! Continue reading for details about this release, and please share any feedback you have!
+
+## Notable Changes
+
+### Stack Changes
+
+* Chromium `87.0.4280.47`
+    * [New in Chrome 86](https://developers.google.com/web/updates/2020/10/nic86)
+    * [New in Chrome 87]()
+* Node.js `12.18.3`
+    * [Node 12.18.3 blog post](https://nodejs.org/en/blog/release/v12.18.3/)
+    * [Node 12.7.0 blog post](https://nodejs.org/en/blog/release/v12.17.0/)
+* V8 `8.7`
+    * [V8 8.6 blog post](https://v8.dev/blog/v8-release-86)
+    * [V8 8.7 blog post](https://v8.dev/blog/v8-release-87)
+
+### Highlight Features
+
+* Support for Apple Silicon: On November 10, Apple announced their [new M1 chips, which will be included in their upcoming hardware](https://www.apple.com/newsroom/2020/11/apple-unleashes-m1/). As of Electron 11, Electron will be shipping separate versions of Electron for Intel Macs (x64) and Apple's upcoming M1 hardware (arm64). You can learn more about how to get your Electron app [running on Apple's M1 hardware here.](https://www.electronjs.org/blog/apple-silicon) [#24545](https://github.com/electron/electron/pull/24545)
+* Added V8 crash message and location information to crashReport parameters. [#24771](https://github.com/electron/electron/pull/24771)
+* Fixed memory leak on macOS when using `dialog.showMessageBox` API. [#26099](https://github.com/electron/electron/pull/26099)
+* Improved the performance of sending JS primitives over the context bridge. [#24531](https://github.com/electron/electron/pull/24531)
+
+See the [11.0.0 release notes](https://github.com/electron/electron/releases/tag/v11.0.0) for a full list of new features and changes.
+
+## Breaking Changes
+
+* Removed experimental APIs: `BrowserView.{fromId, fromWebContents, getAllViews}` and the `id` property of `BrowserView`. [#23578](https://github.com/electron/electron/pull/23578) 
+* Removed the undocumented `desktopCapturer.getMediaSourceIdForWebContents()` API. [#25455](https://github.com/electron/electron/pull/25455)
+
+More information about future changes can be found on the [Planned Breaking Changes](https://github.com/electron/electron/blob/master/docs/breaking-changes.md) page.
+
+## API Changes
+
+* Added support for suspend and resume events to macOS and Windows. [#24254](https://github.com/electron/electron/pull/24254), [#24251](https://github.com/electron/electron/pull/24251)
+* Added language for removal or modification of experimental APIs. [#25222](https://github.com/electron/electron/pull/25222)
+
+### Deprecated APIs
+
+The following experimental APIs are now deprecated or removed:
+
+* Removed experimental APIs: `BrowserView.{fromId, fromWebContents, getAllViews}` and the `id` property of `BrowserView`. [#23578](https://github.com/electron/electron/pull/23578) 
+* Removed the undocumented `desktopCapturer.getMediaSourceIdForWebContents()` API. [#25455](https://github.com/electron/electron/pull/25455)
```

Comment:
Do these need to be called out twice?

Reply:
No, that's probably unnecessary - I'll leave these as is in Breaking Changes, but remove them here